spec/functional/helpers/data-helper.ts

```diff
@@ -25,58 +25,58 @@ interface SampleEntity {
+export class SampleJsonEntity extends SampleEntity { }
 
-export function createHashEntitySchema() : Schema<SampleHashEntity> {
+export function createHashEntitySchema(): Schema<SampleHashEntity> {
   return createSchemaOfType<SampleHashEntity>(SampleHashEntity, 'HASH');
 }
 
-export function createChangedHashEntitySchema() : Schema<SampleHashEntity> {
+export function createChangedHashEntitySchema(): Schema<SampleHashEntity> {
   return createSchemaOfType<SampleHashEntity>(SampleHashEntity, 'HASH', 'sample-hash-entity');
 }
 
-export function createJsonEntitySchema() : Schema<SampleJsonEntity> {
+export function createJsonEntitySchema(): Schema<SampleJsonEntity> {
   return createSchemaOfType<SampleJsonEntity>(SampleJsonEntity, 'JSON');
 }
 
-export function createChangedJsonEntitySchema() : Schema<SampleJsonEntity> {
+export function createChangedJsonEntitySchema(): Schema<SampleJsonEntity> {
   return createSchemaOfType<SampleJsonEntity>(SampleJsonEntity, 'JSON', 'sample-json-entity');
 }
 
-function createSchemaOfType<TEntity extends Entity>(ctor: EntityConstructor<TEntity>, dataStructure: SearchDataStructure, prefix?: string) : Schema<TEntity> {
+function createSchemaOfType<TEntity extends Entity>(ctor: EntityConstructor<TEntity>, dataStructure: SearchDataStructure, prefix?: string): Schema<TEntity> {
   return new Schema<TEntity>(
     ctor, {
-      aString: { type: 'string' },
-      anotherString: { type: 'string' },
-      someText: { type: 'text', sortable: true },
-      someOtherText: { type: 'text', sortable: true },
-      aNumber: { type: 'number', sortable: true },
-      anotherNumber: { type: 'number', sortable: true },
-      aBoolean: { type: 'boolean' },
-      anotherBoolean: { type: 'boolean' },
-      aPoint: { type: 'point' },
-      anotherPoint: { type: 'point' },
-      aDate: { type: 'date', sortable: true },
-      anotherDate: { type: 'date', sortable: true },
-      someStrings: { type: 'string[]' },
-      someOtherStrings: { type: 'string[]' }
-    }, {
-      prefix,
-      dataStructure
-    });
+    aString: { type: 'string' },
+    anotherString: { type: 'string' },
+    someText: { type: 'text', sortable: true },
+    someOtherText: { type: 'text', sortable: true },
+    aNumber: { type: 'number', sortable: true },
+    anotherNumber: { type: 'number', sortable: true },
+    aBoolean: { type: 'boolean' },
+    anotherBoolean: { type: 'boolean' },
+    aPoint: { type: 'point' },
+    anotherPoint: { type: 'point' },
+    aDate: { type: 'date', sortable: true },
+    anotherDate: { type: 'date', sortable: true },
+    someStrings: { type: 'string[]' },
+    someOtherStrings: { type: 'string[]' }
+  }, {
+    prefix,
+    dataStructure
+  });
 }
 
 export async function loadTestHash(client: Client, key: string, data: SampleEntityData) {
 
-  let command: string[] = [];
+  const command: string[] = [];
 
-  for (let field in data) {
-    let value = (data as any)[field];
+  Object.keys(data).forEach(field => {
```

Comment:
Probably less important in the tests. Fine, of course. Just not essential here. ;)

Reply:
i was editing them all and i realised i was already on the tests, but since i had that done i didnt reverse it 🤣